packages/web/src/components/leaderboard/leaderboard-table/tablet-leaderboard-table/TabletLeaderboardTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { TABLET_LEADERBOARD_TD_WIDTH } from "@constants/skeleton.constant";
+import { LeaderModel } from "@models/leaderboard/leader-model";
+import LeaderboardTableRow from "../../leaderboard-table-row/LeaderboardTableRow";
+
+export default function TabletLeaderboardTable({
+  me,
+  leaders,
+  isMobile,
+}: {
+  me?: LeaderModel;
+  leaders: LeaderModel[];
+  isMobile: boolean;
+}) {
+  return (
+    <>
+      {me && (
+        <LeaderboardTableRow
+          item={me}
+          tdWidths={TABLET_LEADERBOARD_TD_WIDTH}
+          isMobile={isMobile}
+          me={true}
+        />
+      )}
```

Comment:
Is there a reason you didn't pass `me` directly?

Reply:
I think you got confused because of the variable names here. So actually `me` is not a boolean but an object, So I cannot pass `me` as a prop because LeaderboardTableRow component takes me as a boolean and not an object, but it takes item as an `object` so that's where I pass `me` object. 

I get how it made you confused, it was such poor naming and I will fix the variable names over here.